spec/features/admin/admin_views_all_users_spec.rb

```diff
@@ -5,18 +5,8 @@
     user = create(:user, admin: true)
     log_in_as user
 
-    visit users_path
+    visit admin_users_path
 
-    expect(page).to have_text(I18n.t("users.index.title"))
-  end
-
-  scenario "unsuccessfully" do
-    user = create(:user)
-    log_in_as user
-
-    visit users_path
-
-    expect(page).to have_text(I18n.t("users.show.welcome"))
-    expect(page).to have_text(I18n.t("users.index.require_admin"))
+    expect(page).to have_text(I18n.t("admin.users.index.title"))
```

Comment:
You should assert here that you can see both admin and user details. And you might want to check that they are displayed in the order you wish them to be displayed.

Reply:
I have done the assertions for admin and user details. I am struggling to find a way to test the sort order, I have tried `http://launchware.com/articles/acceptance-testing-asserting-sort-order` but can't get it to work for a ul. Do you have any suggestions on where to get more info on this as I would like to test it?

Reply:
I've added a spec for the correct order in `spec/models/user_spec.rb`
Therefore, we don't need to check the order here. Just that both admin and non-admin users appear on the page.

So what I would do is: create 1 admin, and 1 user. Then check that both their names appear on the page.
But I would check that "John" and "Will are on the page, rather than `admin.name`

Reply:
Thanks Elle, really great spec. done in sha: 9bd5721